packages/terra-status-view/src/StatusView.jsx

```diff
@@ -28,6 +28,7 @@ const propTypes = {
 
   /**
    * Display a custom glyph. Overrides a variant's default glyph.
+   * Set `focusable=false` for svg element used as `customGlyph`.
```

Comment:
This comment might not be necessary anymore, but it's not harmful to leave it if users are using a a custom SVG